docassemble/ALKilnTests/data/questions/test_loops.yml

```
@@ -0,0 +1,66 @@
+metadata:
+  title: Loop tests
+  description: Test loops like as with .there_is_another and .target_number
+---
+# Necessary to tell us what the sought var is on each page
+# Every interview that wants testing will need to have an element like this
+default screen parts:
+  post: |
+    <div data-variable="${ encode_name(str( user_info().variable )) }" id="trigger" aria-hidden="true" style="display: none;"></div>
+---
+objects:
+  - there_are_any_people: DAList.using(object_type=Individual)
+  - there_is_another_people: DAList.using(object_type=Individual, there_are_any=True)
+  - target_people: DAList.using(object_type=Individual, ask_number=True)
+---
+mandatory: True
+code: |
+  there_are_any_people.gather()
+  there_is_another_people.gather()
+  target_people.gather()
+  end1
```

```suggestion
  end
```

As per https://github.com/SuffolkLITLab/ALKiln/pull/709#discussion_r1228171965.